plugins/mcreator-localization/help/default/dimension/generate_aquifers.md

```diff
@@ -0,0 +1 @@
+If this parameter is enabled, aquifers will generate in this dimension.
```

Comment:
I would explain a bit on what "aquifers" are, because this term may be an unknown for new users